### PR TITLE
release(v0.3.0-alpha.0): prepare release

### PR DESCRIPTION
**Pull request by rsmitty**
This is the official v0.3.0-alpha.0 release.

Signed-off-by: Spencer Smith <robertspencersmith@gmail.com>

### CODE REVIEW

File: hack/chglog/config.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@ style: github
 template: CHANGELOG.tpl.md
 info:
   title: CHANGELOG
-  repository_url: https://github.com/talos-systems/talos
+  repository_url: https://github.com/talos-systems/sidero
```
Comment (smira) [inline hack/chglog/config.yml:5]: not for this PR, but I would love us to switch to `release-tool` like Talos

**Review by smira — DISMISSED** [review]

File: hack/chglog/config.yml (hunk 1 above)
Comment (rsmitty) [reply hack/chglog/config.yml:5]: oh yeah, totally. I had some trouble with the release script this time. will add that as a 0.3 task.

**Pull request merged**